### PR TITLE
Dt/feature/add a post endpoint and notification test

### PR DESCRIPTION
**Pull request by DaTanUmanitoba**

### CODE REVIEW

File: backend/controller/post-controller.js [hunk 1]
```diff
@@ -55,7 +55,7 @@ const getPostByID = async (req, res) => {
   try {
     const post = await postService.getPostByID(post_id);
     const result = await getPostsInfo([post]);
-    res.json(Result.success(result));
+    res.json(Result.success(result[0]));
```
Comment (orangeboy55555) [inline backend/controller/post-controller.js:58]: Can you remove the [0] on line 58? I fixed this problem on my branch by changing the test.

**Review by orangeboy55555 — CHANGES_REQUESTED** [review]
All good!

File: backend/controller/post-controller.js (hunk 1 above)
Comment (DaTanUmanitoba) [reply backend/controller/post-controller.js:58]: OK.

**Review by orangeboy55555 — APPROVED** [review]

**Review by ShahzaibParacha — APPROVED** [review]
Okay to merge

**Pull request merged**